hr_employee/models/contract_enterprise.py

```diff
@@ -37,12 +37,54 @@ def _default_sign_date(self):
             ], string='Status', default='new')
 
     total_salary = fields.Float(string='Total Salary', compute='_compute_total_salary', store=True)
-
+    # cách 1
+    role_name = fields.Char(compute='_compute_get_role_name', store=True, string='Role Name', readonly=False)
+    # level_name = fields.Char(compute='_compute_get_level_name', store=True, string='Level Name', readonly=False)
+    # cách 2
+    level_name = fields.Char(related="employee_id.work_experiences_ids.level_id.name", store=True, string='Level Name', readonly=False)
```

Comment:
Trong trường hợp work_experiences_ids có nhiều bản ghi thì hệ thống sẽ xử lý ntn?

Reply:
đối với case này em đang hiểu bussines sẽ là 1 employee sẽ chỉ có nhiều record kinh nghiệm làm việc
+ em có 2 solution:
1, solution em đang thực hiện ở trên là em cho hiển thị work_experiences latest tức là lấy record mới nhất được thêm vào sau cùng
2, vì ở view contract chỉ hiển thị 3 trường là job_name, level_name, role_name nên em có thể sửa theo solution là cho hiển thị diện dạng dropdown list để hiển thị toàn bộ role_name, job_name và level_name của employee đó